tvb_documentation/manuals/UserGuide/DataExchange.rst

```diff
@@ -194,6 +197,23 @@ In correlation with export operations, |TVB| interface allows import of data in
 |TVB| data type. Depending on the uploaded file format, imported data can be as
 follows:
 
+File Format
+~~~~~~~~~~~
+
+1. If user uploads a ZIP file, the system automatically assumes a datatype group
+   must be imported and then process the file accordingly. More specifically, it
+   tries to find an XML file, within the ZIP file, describing the operation(s)
+   that generated the data types and the list of HDF5 files for each datatype.
```

Comment:
This paragraph should be corrected to describe the current flow, where we have H5 view model instead of XML and when importing a ZIP tvb has to determine whether it contains a DT group or a DT with links

Reply:
I think it actually looks if there are directories inside the zip file and if there are, it takes it as a DatatypeGroup import, otherwise it is an import with links.